src/components/playground/CommentContent.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+import ReactEmoji from 'react-emoji';
+
+@connect(state => state.playground)
+@Radium
+class CommentContent extends React.Component {
+
+  filterContent(content) {
+    /*
+      This pattern searches for tags within comment content,
+      those tags relate to the Playground's settings as in:
+
+      {moderation.muting}...something...{/moderation.muting}
+
+      So we can show some mutations inside the content.
+    */
+    var tagSearchPattern = /{(.+)}(.*){\/(\1)}/;
+    var matches = content.match(tagSearchPattern);
+    var filteredContent = content;
+    if (matches) {
+      var togglerGroupAndKey = matches[1].split(".");
+      if (this.props.togglerGroups[togglerGroupAndKey[0]].togglers[togglerGroupAndKey[1]].status) {
+        filteredContent = content.replace("{" + matches[1] + "}" + matches[2] + "{/" + matches[1] + "}", matches[2]);
```

Comment:
Hmmmm not sure, unless things get too cluttered, I tend to prefer the readability of straight concatenation.